dev-util/hut/hut-0.5.0.ebuild

```diff
@@ -21,7 +21,6 @@ SLOT="0"
 
 KEYWORDS="~amd64"
 
-
 DEPEND="${RDEPEND}"
 BDEPEND="app-text/scdoc"
```

Comment:
This is incorrect. The policy states that:
- `USE=man` must not be used, since the package must always install a manpage
- prebuilding it and shipping it as an additional source is a *good alternative* to requiring "inconvenient" additional dependencies

scdoc isn't an inconvenient additional dependency. I don't think it's necessary to prebuild it, as the package currently doesn't violate the policy document.

Reply:
@eli-schwartz It seems I misunderstood part of the policy. Your second bullet point clarified it for me, thanks!